app.js

```diff
@@ -17,6 +17,9 @@ app.get("/api/articles", getAllArticles)
 
 app.get("/api/articles/:article_id/comments", getComments)
 
+app.use(express.json())
```

Comment:
Really good demonstration of TDD principles to include this only now that we _need it_ to pass the test 🥇